dts/bindings/gpio/intel,ehl-gpio.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Intel Elkhart Lake GPIO node
```

Comment:
Do you really need to add any new DTS yaml files now that we use a single driver? Seems to me like you should just make the existing APL GPIO yaml file generic the same way you made the driver generic.